Framework/PythonInterface/plugins/algorithms/WorkflowAlgorithms/ElasticWindowMultiple.py

```diff
@@ -318,8 +320,9 @@ def _get_sample_units(self, ws_name):
         if self._sample_log_name in run:
             # Look for sample unit in logs in workspace
             tmp = run[self._sample_log_name].value
-            value_action = {'last_value': lambda x: x[len(x) - 1],
-                            'average': lambda x: x.mean()}
+            value_action = {'last_value': lambda x: x[-1],
+                            'average': lambda x: x.mean(),
+                            'lowest_value': lambda x: np.amin(x)}
```

Comment:
We're just waiting to see whether this pull request is relevant - there was potentially miscommunication from Spencer.